users/templates/user_site/home.html

```diff
@@ -27,6 +27,7 @@
         $(document).ready(function(){
             const TIMEOUT = 10000;
 
+            vote_url = "{% url 'meeting_vote' meeting_id=1 %}";
```

Comment:
what is the point of the meeting_id=1 ? what does '1' represents?

Reply:
Meeting vote needs a meeting if variable otherwise the url will not work,
I haven't seen a way to insert a variable into the url statement and in this way I enter a default variable and then cut it using us when I use it